src/model_signing/signing/in_toto.py

```diff
@@ -167,6 +129,7 @@ class DigestsIntotoPayload(IntotoPayload):
     """
 
     predicate_type: Final[str] = "https://model_signing/Digests/v0.1"
```

Comment:
are we still happy with this predicate string?

Reply:
I'll change it when we get the new format.